lib/vanagon/driver.rb

```diff
@@ -74,6 +74,8 @@ def pick_engine(options) # rubocop:disable Metrics/PerceivedComplexity
     end
 
     def load_engine_object(engine_type, platform, target)
+      engine_type = 'base' if target
```

Comment:
It looks like the bug is in this method https://github.com/puppetlabs/vanagon/commit/eaaea24ed314b00759154f6d86abe5ef6df6234f#r44961052 which i think could use a refactor anyway to be easier to reason about.